devops/actions/aws-ec2/aws-ec2.js

```diff
@@ -87,7 +93,7 @@ async function start(param_type, param_label, param_ami, param_spot, param_disk,
           const items = ec2disk.split(':');
           params.BlockDeviceMappings = [ {DeviceName: items[0], Ebs: {VolumeSize: items[1]}} ];
         }
-        const result = await ec2.runInstances(params).promise();
+        const result = await ec2.runInstances(params);
```

Comment:
I would also replace this to match v3: https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/client/ec2/command/RunInstancesCommand/